### ISSUE TITLE
Register activity for poll change & vote

### ISSUE THREAD

**Issue by CoreyBurger** [nextcloud/polls#804]
The activity stream doesn't include any of the poll creation/deletion/closure or the votes.

How the activity app works:
https://github.com/nextcloud/activity/blob/master/docs/create.md
and 
https://github.com/nextcloud/activity/blob/master/docs/provider.md

**Comment by dartcafe** [comment]
I already started working on that, but it is more complax as it seems. so, no DOA ATM.

**Comment by CoreyBurger** [comment]
If you push to a branch, I'd be happy to take a look myself over the next week

**Comment by dartcafe** [comment]
Have no senseful branch. Just played around a little bit and let it die for the moment.

**Comment by Firerazer** [comment]
Hello there,
I am using your poll app, and not having notifications is quite a problem as my users have a tendency to miss new polls because they don't take time to check all tabs when they connect, but only check activity to see what's new.
Is there any new idea for it? I know a lot of other apps are using it successfully, cannot you check how they are doing it?
Thanks in advance for this good work already, hope you will continue to maintain it and update this!

**Comment by dartcafe** [comment]
This is something, which needs the integartion of the notification app because users need to manage their subscriptions to apps by themself. Unfortnunately my time will run short in the next months, so I fear I will not have the time to get into the doocs and development. 

@v1r0x Are you able to do this?

**Comment by Firerazer** [comment]
Hello,
I just see that I mix without paying attention notification and activity. We do not have notification system active, we just refer to the activity tab that is native (I think?) into nextcloud. On all apps that I use, yours is the only one that do not hva anything tracking the activity.
Thanks,

**Comment by v1r0x** [comment]
@dartcafe The docs from OP look not that hard to follow. I can try to work on it. But no promises as I don't have that much time for it. But I'll create a new branch so everyone can track the progress and jump on development.

Any suggestions on what should appear in the activity tab?

**Comment by Firerazer** [comment]
On the activity tab:
- New poll creation
- Poll response added
- Poll closed/arrived in due date
- Poll options edited
